lib/jekyll-import/importers/tumblr.rb

```diff
@@ -62,7 +62,6 @@ def self.process(options)
         end
       end
 
-      private
```

Comment:
same comment here about `private_class_method` as above

Reply:
it seems to breaks the tests, leaving this public for now.

Reply:
Couldn't figure out how to add a private_class_method without breaking the tests, maybe it needs a `send` method in the tests?